### PR TITLE
OCPBUGS-4771: Decapitalize the first letter of ContainerLogMaxSize

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11, 4.12

Issue: https://issues.redhat.com/browse/OCPBUGS-4771

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kcarmichael08** [comment]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/create-a-containerruntimeconfig-crd.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ The CRI-O flag is applied on the cgroup of the container, while the Kubelet flag
 
 * **Log level**: The `logLevel` parameter sets the CRI-O `log_level` parameter, which is the level of verbosity for log messages. The default is `info` (`log_level = info`). Other options include `fatal`, `panic`, `error`, `warn`, `debug`, and `trace`.
 * **Overlay size**: The `overlaySize` parameter sets the CRI-O Overlay storage driver `size` parameter, which is the maximum size of a container image.
-* **Maximum log size**: Setting the maximum log size in the `ContainerRuntimeConfig` is expected to be deprecated. If a maximum log size is required, it is recommended to use the `ContainerLogMaxSize` field in the `KubeletConfig` CR instead.
+* **Maximum log size**: Setting the maximum log size in the `ContainerRuntimeConfig` is expected to be deprecated. If a maximum log size is required, it is recommended to use the `containerLogMaxSize` field in the `KubeletConfig` CR instead.
```
Comment (opayne1) [inline modules/create-a-containerruntimeconfig-crd.adoc:27]: Changes from peer review side lgtm. I see your comment in the bug that you double checked it.

**Pull request merged**